doc/android-build.md

```diff
@@ -0,0 +1,37 @@
+# Building binaries of LibIndy for Android
+
+**Not ready for production use! Not fully tested.**
+
+## Prerequisites
+
+- Docker
+
+## How to build.
+- Goto `indy-sdk/libindy/build_scripts/android` folder
+- Run `build.dependencies.locally.sh`
```

Comment:
> Run `build.dependencies.locally.sh`

Will it build dependencies for all supported architectures?

Reply:
Yes. The script will build for x86, arm and arm64 architectures.